WobblyStealthGame/Assets/Scripts/Input/ActiveRagdoll/BalanceModule.cs

```diff
@@ -4,12 +4,21 @@
 
 public class BalanceModule : Module
 {
-    public float customTorsoAngularDrag = 0.05f;
+    private enum Mode {
+        Torgue,
```

Comment:
```suggestion
    private enum Mode
    {
        Torgue,
```
Change torgue -> torque